syntax = "proto3";
import "proto/system.proto";

message Metadata {
  string type = 1;
  string value = 2;
}

message PointOfInterest {
  string id = 1;
  string system_name = 2;
  string planet_name = 3;
  Coordinates coordinates = 4;
  string discovered_by = 5;
  string type = 6;
  repeated Metadata metadata = 7;
}
